packages/zarf-registry/chart/values.yaml

```diff
@@ -54,7 +54,43 @@ autoscaling:
   maxReplicas: 5
   targetCPUUtilizationPercentage: 80
 
+caBundle: ""
+## One or more concatenated certificates
+## Will be mounted to /etc/ssl/certs/ca-certificates.crt
+# caBundle: |
+#   # Root CA 1
+#   -----BEGIN CERTIFICATE-----
+#   ...
+#   -----END CERTIFICATE-----
+#   # Intermediate CA 1
+#   -----BEGIN CERTIFICATE-----
+#   ...
+#   -----END CERTIFICATE-----
+#   # Root CA 2
+#   -----BEGIN CERTIFICATE-----
+#   ...
+#   -----END CERTIFICATE-----
+
 extraEnvVars: []
 ## Additional ENV variables to set
 # - name: REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY
 #   value: "/var/lib/example"
+
+extraVolumes: []
+## Additional volumes to the pod.
+# - name: my-custom-volume
+#   configMap:
+#     name: my-custom-configmap
+
+extraVolumeMounts: []
+## Additional volumeMounts to the registry container.
+#  - mountPath: /my/custom/file/path/
+#    name: my-custom-volume
+
+extraConfigMaps: []
+## Additional configMaps to create in the release namespace
+# - name: my-custom-configmap
+#   data:
+#     my-file.txt: |
+#       foo
+#       bar
```

Comment:
Just to keep this simpler I would rather not expose these unless there is a common reason to do so (people can always make a custom init package if they _really_ need something special).  The default init package is geared toward IT admins who have less k8s experience and these options may end up being confusing (since they will show up in package inspection and are settable at deploy time).  We are working to get these personas published as well for external contributors to see (like GitLab does) but haven't completed that yet.

Reply:
[ashton_gru.pdf](https://github.com/defenseunicorns/zarf/files/12579047/ashton_gru.pdf)